app/src/main/java/org/stepic/droid/preferences/SharedPreferenceHelper.java

```diff
@@ -840,4 +855,14 @@ private boolean getBoolean(PreferenceType preferenceType, String key, boolean de
         return context.getSharedPreferences(preferenceType.getStoreName(), Context.MODE_PRIVATE)
                 .getBoolean(key, defaultValue);
     }
+
+    private float getFloat(PreferenceType preferenceType, String key, float defaultValue) {
+        return context.getSharedPreferences(preferenceType.getStoreName(), Context.MODE_PRIVATE)
+                .getFloat(key, defaultValue);
+    }
+
+    private float getFloat(PreferenceType preferenceType, String key) {
```

Comment:
Usually methods like that calls parent method with default argument instead of duplicating its body